frontend/dashboard/pages/PageLayout/PageLayout.test.tsx

```diff
@@ -64,10 +63,45 @@ describe('PageLayout', () => {
 
   test('should redirect to root if user does not have access to selected context', async () => {
     (useParams as jest.Mock).mockReturnValue({
-      selectedContext: 'test',
+      selectedContext: 'testinvalidcontext',
     });
     renderWithMockServices();
     expect(mockedNavigate).toHaveBeenCalledTimes(1);
-    expect(mockedNavigate).toHaveBeenCalledWith(DASHBOARD_ROOT_ROUTE);
+    expect(mockedNavigate).toHaveBeenCalledWith(SelectedContextType.Self, expect.anything());
+  });
+
+  test('should redirect to self context if none is defined', async () => {
```

Comment:
What do you think about changing test to it? This way, the sentence will start with "it should...", making it even more readable.